src/Nethermind/Nethermind.Core/Crypto/Keccak.cs

```diff
@@ -261,19 +204,36 @@ public class Keccak : IEquatable<Keccak>, IComparable<Keccak>
         /// </summary>
         public static Keccak MaxValue { get; } = new("0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
 
-        public byte[] Bytes { get; }
```

Comment:
I was thinking of changing this to return a `ReadOnlySpan`. That would've significantly reduce the amount of change.

Reply:
Changed `Span` to `Bytes`